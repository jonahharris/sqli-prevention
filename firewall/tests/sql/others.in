>>> Sada testovacich SQL dotazu: show
# databases
SHOW DATABASES;
SHOW SCHEMAS;
SHOW DATABASES LIKE 'm%';
SHOW DATABASES WHERE Tables_in_test LIKE 'a%';
SHOW SCHEMAS LIKE 'm%';
SHOW SCHEMAS WHERE Tables_in_test LIKE 'a%';
# tables
SHOW TABLES;
SHOW TABLES WHERE Tables_in_test LIKE 'a%';
SHOW FULL TABLES;
# describe
DESCRIBE city;
DESC authors;
DESCRIBE city size;
DESC authors name;
# use
USE db1;
USE db2;
# starts
SELECT * FROM t1 WHERE id = ***;
SELECT * FROM t1 WHERE id = '***';
SELECT * FROM t1 LEFT JOIN t2 ON t1.id = t2.`***`;
SELECT * FROM t1 LEFT JOIN t2 ON t1.id = `***`;
SELECT *** FROM t2;
SELECT '***' FROM t2;
SELECT `***` FROM t2;
SELECT `***`.* FROM t2;
SELECT * FROM `***`;
SELECT * FROM db1.`***`;
SELECT * FROM `***`.tb1;
SELECT * FROM t1 LEFT JOIN t2 USING (`***`);
SELECT * FROM t1 ORDER BY `***` ASC;
SELECT * FROM t1 ORDER BY '***' ASC;
SELECT * FROM t1 ORDER BY i ***;
