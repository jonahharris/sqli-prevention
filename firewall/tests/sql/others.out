s>>> Sada testovacich SQL dotazu: show a dalsi
# databases
SHOW DATABASES;STMT;1;0;
SHOW DATABASES;STMT;1;0;
STRING '***';LIKE;SHOW DATABASES;STMT;1;0;
NAME Tables_in_test;STRING '***';LIKE;WHERE;SHOW DATABASES;STMT;1;0;
STRING '***';LIKE;SHOW DATABASES;STMT;1;0;
NAME Tables_in_test;STRING '***';LIKE;WHERE;SHOW DATABASES;STMT;1;0;
# tables
SHOW TABLES 0;STMT;1;0;
NAME Tables_in_test;STRING '***';LIKE;WHERE;SHOW TABLES 0;STMT;1;0;
SHOW TABLES 1;STMT;1;0;
# describe
TABLE city;DESCRIBE;STMT;1;0;
TABLE authors;DESCRIBE;STMT;1;0;
TABLE city;COLUMN size;DESCRIBE;STMT;1;0;
TABLE authors;COLUMN name;DESCRIBE;STMT;1;0;
# use
DATABASE db1;USE;STMT;1;0;
DATABASE db2;USE;STMT;1;0;
# starts
SELECTALL;TABLE t1;NAME id;STRING '***';CMP 4;WHERE;SELECT 0 1 1;STMT;1;0;
SELECTALL;TABLE t1;NAME id;STRING '***';CMP 4;WHERE;SELECT 0 1 1;STMT;1;0;
SELECTALL;TABLE t1;TABLE t2;FIELDNAME t1.id;FIELDNAME t2.***;CMP 4;ONEXPR;JOIN 301;SELECT 0 1 1;STMT;1;0;
SELECTALL;TABLE t1;TABLE t2;FIELDNAME t1.id;NAME ***;CMP 4;ONEXPR;JOIN 301;SELECT 0 1 1;STMT;1;0;
STRING '***';TABLE t2;SELECT 0 1 1;STMT;1;0;
STRING '***';TABLE t2;SELECT 0 1 1;STMT;1;0;
NAME ***;TABLE t2;SELECT 0 1 1;STMT;1;0;
FIELDNAME ***.*;TABLE t2;SELECT 0 1 1;STMT;1;0;
SELECTALL;TABLE ***;SELECT 0 1 1;STMT;1;0;
SELECTALL;TABLE db1.***;SELECT 0 1 1;STMT;1;0;
SELECTALL;TABLE ***.tb1;SELECT 0 1 1;STMT;1;0;
SELECTALL;TABLE t1;TABLE t2;COLUMN ***;USING 1;JOIN 301;SELECT 0 1 1;STMT;1;0;
SELECTALL;TABLE t1;NAME ***;ORDERBY 1;ORDERBYLIST 1;SELECT 0 1 1;STMT;1;0;
SELECTALL;TABLE t1;STRING '***';ORDERBY 1;ORDERBYLIST 1;SELECT 0 1 1;STMT;1;0;
SELECTALL;TABLE t1;NAME i;ORDERBY 3;ORDERBYLIST 1;SELECT 0 1 1;STMT;1;0;
